src/integrations/ga/SplitToGa.js

```diff
@@ -5,18 +5,15 @@ const log = logFactory('splitio-split-to-ga');
 
 class SplitToGa {
 
-  // Default filter function. It accepts all Split events and impressions.
-  static defaultFilter() { return true; }
-
   // Default mapper function.
   static defaultMapper({ type, payload }) {
     switch (type) {
       case SPLIT_IMPRESSION:
         return {
           hitType: 'event',
           eventCategory: 'split-impression',
-          eventAction: payload.impression.feature,
-          eventLabel: payload.impression.treatment,
+          eventAction: 'Evaluate ' + payload.impression.feature,
+          eventLabel: 'Treatment ' + payload.impression.treatment + ' Label ' + payload.impression.label,
```

Comment:
Let's make it: `'Treatment: ' + payload.impression.treatment + '. Targeting rule: ' + payload.impression.label + '.',`